tools/splat.yaml

```diff
@@ -909,6 +829,7 @@ segments:
     - [0x7E3700, "c"]
     - [0x7e4d00, "bin"]
   - name: world/area_mac/machi/
+    ld_name: machi
```

Comment:
I know this is technically a splat thing and I should have caught it there, but I prefer the LD stuff to all be handled behind the scenes as much as possible. What's the reason for this option? What would happen without it?

Reply:
It's used for the asset table, which isn't in this PR. It allows us to refer to this as `ROM_machi` etc. (see the generated include/ld_addrs.h).

Reply:
yeah I feel like we should probably make the ld names different, but again my bad for not really catching that in the splat pr. I kinda knew I'd need to see it in practice to have an opinion.